src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,108 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_VOLUME = 10;
+    private int size = 0;
```

Comment:
`int` variables is zero by default, no need to initialize them explicitly
```suggestion
    private int size;
```